src/pal/tools/gen-buildsys-gcc.sh

```diff
@@ -177,12 +155,10 @@ cmake \
   "-DCMAKE_LINKER=$gcc_link" \
   "-DCMAKE_NM=$gcc_nm" \
   "-DCMAKE_RANLIB=$gcc_ranlib" \
-  "-DCMAKE_OBJCOPY=$gcc_objcopy" \
   "-DCMAKE_OBJDUMP=$gcc_objdump" \
   "-DCMAKE_BUILD_TYPE=$buildtype" \
   "-DCMAKE_EXPORT_COMPILE_COMMANDS=1 " \
   "-DCLR_CMAKE_ENABLE_CODE_COVERAGE=$code_coverage" \
-  "-DCLR_CMAKE_COMPILER=GNU" \
```

Comment:
Don't we need this? There are CMakeLists.txt files that look at the GNU string in CLR_CMAKE_COMPILER setting.

Reply:
There we use `CMAKE_C_COMPILER_ID STREQUAL Clang` instead of a custom property.

Reply:
Ah, I see. 

We used (CMAKE_CXX_COMPILER_ID STREQUAL "Clang"). It would be good to be consistent here.

This is my search result:

cross/toolchain.cmake:# This gets called with CLR_CMAKE_COMPILER set on the first invocation
cross/toolchain.cmake:# Use environment variable to keep CLR_CMAKE_COMPILER around.
cross/toolchain.cmake:if (NOT DEFINED CLR_CMAKE_COMPILER)
cross/toolchain.cmake:  set(CLR_CMAKE_COMPILER $ENV{CLR_CMAKE_COMPILER})
cross/toolchain.cmake:  set(ENV{CLR_CMAKE_COMPILER} ${CLR_CMAKE_COMPILER})
cross/toolchain.cmake:if (CLR_CMAKE_COMPILER STREQUAL "Clang")
cross/toolchain.cmake:if (CLR_CMAKE_COMPILER STREQUAL "Clang")
src/pal/tools/gen-buildsys-clang.sh:  "-DCLR_CMAKE_COMPILER=Clang" \
src/pal/tools/gen-buildsys-gcc.sh:  "-DCLR_CMAKE_COMPILER=GNU" \